index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    
+    
+    <header>
+        <div class="time">
+            <img src="images/Time.png" alt="hamburger">
+            <img src="images/Group 3.png" alt="hamburger">
+        </div>
+        <div class="portfolio-header">
+            <a class="logo">My Logo</a>
```

Comment:
[OPTIONAL]
Good job wrapping your logo in `<a>` tag. Kindly add the` href="/"` should add as well and route it to the index page such that in case you have multiple sections, clicking the logo will take you to the index page which offers a good user experience
e.g 
`<a class="logo"  href="/">My Logo</a>`